README.md

```diff
@@ -5,12 +5,36 @@
 ### Introduction
 
 The service-catalog project is in incubation to bring integration with service
-brokers to the Kubernetes ecosystem via the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
-A service broker is an endpoint that manages a set of services.  The end-goal of
-the service-catalog project is to provide a way for Kubernetes users to consume
-services from brokers and easily configure their applications to use those
-services.
+brokers to the Kubernetes ecosystem via the [Open Service Broker
+API](https://github.com/openservicebrokerapi/servicebroker). A service broker
+is an endpoint that manages a set of services.  The end-goal of the service-
+catalog project is to provide a way for Kubernetes users to consume services
+from brokers and easily configure their applications to use those services,
+without needing detailed knowledge about how those services are created /
+managed.
+
+As an example:
+
+Say that a distributed application needs a message broker that hosts a set of
+queues/topics.  The service-catalog allows Kubernetes applications to consume
+services like message brokers that exists _somewhere_ in a simple way:
+
+1.  A user wanting to consume a service browses a list of available services
+    in the catalog
+2.  The user asks for a new instance of that service to be _provisioned_
+
+     _Provisioning_ means that the broker somehow creates a new instance of a
+    service.  This could mean creating a new set of Kubernetes resources in another
```

Comment:
> This could mean creating a new set of Kubernetes resources in another  namespace in the same Kubernetes cluster as the consumer or a different cluster, creating a new deployment in another orchestration system entirely, or even creating a new tenant in a multi-tenant SaaS system. 

Would it be good to break this sentence apart?